code/game/machinery/cryo.dm

```diff
@@ -193,6 +193,11 @@
 	var/is_on = on && operable()
 	icon_state = "[icon_state]-[is_on ? "on" : "off"]-[occupant ? "occupied" : "empty"]"
 
+/obj/structure/machinery/cryo_cell/Destroy()
+	if(occupant)
+		go_out()
+	..()
```

Comment:
```suggestion
	. = ..()
```

Reply:
better as `return ..()` but yes this is the problem.

Reply:
Functionally identical, I just grabbed the nearest example, but yes.